### PR TITLE
glTF new drop and updates

### PR DESCRIPTION
**Pull request by keveleigh**
Overview
---
This update includes the latest drop from https://github.com/KhronosGroup/UnityGLTF. This drop cleans up quite a bit of excess code, as the schema scripts are now found in the GLTFSerialization DLL.

It removes a script I created (GLTFComponentStreamingAssets) in favor of new support in this drop.

It also replaces our previous (incorrect, as we are not using specular materials) use of the Unity Standard (Specular) shader in favor of the project's GLTFStandard shader.

Changes
---
Related to #1644

### CODE REVIEW

File: Assets/HoloToolkit-Examples/Input/Materials/HoloToolkit_Focusable.mat [hunk 1]
```diff
@@ -8,7 +8,8 @@ Material:
   m_PrefabInternal: {fileID: 0}
   m_Name: HoloToolkit_Focusable
   m_Shader: {fileID: 4800000, guid: f1f3948b20c230044bda31b620ddd37f, type: 3}
-  m_ShaderKeywords: _EMISSION
+  m_ShaderKeywords: _EMISSION _SPECULARHIGHLIGHTS_ON _USEAMBIENT_ON _USEDIFFUSE_ON
```
Comment (StephenHodgson) [inline Assets/HoloToolkit-Examples/Input/Materials/HoloToolkit_Focusable.mat]: Does updating this material required for the gLTF update?

File: Assets/HoloToolkit/Utilities/Scripts/GLTF/Scripts/GLTFComponent.cs.meta [hunk 2]
```diff
@@ -1,11 +1,11 @@
 fileFormatVersion: 2
-guid: 3628db10e32278e499da95887f1bc989
-timeCreated: 1488842155
+guid: 9087a95f22bcee64ebc45f6deec08714
```
Comment (StephenHodgson) [inline Assets/HoloToolkit/Utilities/Scripts/GLTF/Scripts/GLTFComponent.cs.meta]: Will this meta file change break the scene references in other people's projects?

**Comment by DDReaper** [comment]
My only concern is the same as @StephenHodgson regarding the .meta files, should try to retain the original GUID's if we can.  Unless these are all new from the source repo.

A few concerns with the Kronos scripts, but they need to be addressed in the base repo, not here.

File: Assets/HoloToolkit/Utilities/Scripts/GLTF/Scripts/GLTFComponent.cs.meta (hunk 2 above)
Comment (keveleigh) [reply Assets/HoloToolkit/Utilities/Scripts/GLTF/Scripts/GLTFComponent.cs.meta]: This is an update from the source repo. Looks like they changed it there, so I'm leaving this edit here to match it.

File: Assets/HoloToolkit-Examples/Input/Materials/HoloToolkit_Focusable.mat (hunk 1 above)
Comment (keveleigh) [reply Assets/HoloToolkit-Examples/Input/Materials/HoloToolkit_Focusable.mat]: The MotionControllerTest scene's cube is broken without it. I suppose it's not required for the glTF drop itself, but it fixes the main test scene for the glTF models.

**Pull request merged**